modules/observe/adoption-insights/proc-install-adoption-insights.adoc

```diff
@@ -16,35 +16,37 @@ To enable the Adoption Insights plugin, complete the following steps:
 +
 [source,yaml]
 ----
-- package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-adoption-insights:bs_1.35.1__0.0.2!red-hat-developer-hub-backstage-plugin-adoption-insights
-  disabled: false
-  pluginConfig:
-    dynamicPlugins:
-      frontend:
-        red-hat-developer-hub.backstage-plugin-adoption-insights:
-          appIcons:
-            - name: adoptionInsightsIcon
-              importName: AdoptionInsightsIcon
-          dynamicRoutes:
-            - path: /adoption-insights
-              importName: AdoptionInsightsPage
-              menuItem:
+  - package: ./dynamic-plugins/dist/backstage-community-plugin-analytics-provider-segment
+    disabled: true
```

Comment:
@pabel-rh I see we already have this added in another section which talks about disabling the segment provider- https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.5/html-single/red_hat_developer_hub_release_notes/index#developer-preview-rhdhpai-510

So can we remove this block in favour of the above section?
```suggestion

```